spring-cloud-dataflow-server-core/pom.xml

```diff
@@ -141,7 +141,6 @@
 		<dependency>
 			<groupId>com.h2database</groupId>
 			<artifactId>h2</artifactId>
-			<optional>true</optional>
```

Comment:
Could we keep this optional in core and add it back in `spring-cloud-dataflow-server` pom?

Reply:
I am sure we can. This would have a benefit of being propagated transitively to fewer other libs. This makes sense and is a better place to put it, but is there another benefit or angle that this provides (rebuild the server w/o the h2 locally ??).

Reply:
Also, just to clarify, it was originally in core.

Reply:
@jvalkeal - done